### ISSUE TITLE
strange log? Crash on enter inventory/hit a block - any version

### ISSUE THREAD

**Issue by metthelm** [The-1-7-10-Pack/The-1.7.10-Pack#751]
Hi, the game crashes instantly by entering the inventory , or hitting any block.
i saw alot of similar posts, but couldnt find any helpful stuff to solve my problem. i'm very new in setting-up servers and even dont know how to find my real crashlog.....
what i found (on serverside) ist this:

siblings=[], style=Style{hasParent=false, color=null, bold=null, italic=null , underlined=null, obfuscated=null, clickEvent=null, hoverEvent=null

maybe here is the problem?

thx and greetings

**Comment by The-1-7-10-Pack** [comment]
Just try to setup a new item.
The report before said that they set-up the server wrong :)

**Comment by metthelm** [comment]
sorry, i dont really get your answer :( what item do u mean?

**Comment by The-1-7-10-Pack** [comment]
Redownload the server files, unzip them, and relaunch the server.

**Comment by metthelm** [comment]
maybe this is the log u need: http://paste.ubuntu.com/11879285/

i have rented a server for the first time. i downloaded the serverfiles, unzipped them, and uploaded them. the server starts, but when i join, then... see the first post :)

**Comment by The-1-7-10-Pack** [comment]
Yes I saw these logs before, if you don't set up right the server with all the needed files it'll have these problems.

**Comment by metthelm** [comment]
i reload the complete folder and restart the server with the exact same result. do i need any further files ? or what do you mean with: "if you don't set up right the server with all the needed files" ?

**Comment by The-1-7-10-Pack** [comment]
Delete the server and completely redownload everything

**Comment by metthelm** [comment]
i did delete all completely from the PC and reinstall Mc/the pack. but i have the same results...

On serverside, i can delete almost all files from the mainfolder, instead of 2 files called: "start" and "rtoolkit.sh".
i can create folder, but cant delete them ?!?
 i just opened a ticket.
sry for my bad english, its not my native language ;)

**Comment by The-1-7-10-Pack** [comment]
Just

**Comment by The-1-7-10-Pack** [comment]
I meant server side. Get new server files and use them :)

**Comment by metthelm** [comment]
I did! more then once :( there are a few folder on this server (outside the minecraftfolder) which i cantg delete (opned a ticket 4 the support) I figured out, that the server runs java 7, because 98% of all mods, arent compatile witth the 8version... i told them, that i only need the 8th version.  i have to wait, i think. badly, i dont have the 777 on the server :(

thx anyway for your patience and good will to help me :) 
keep on the very good work!!!!
